src/main/webapp/index.html

```diff
@@ -148,8 +147,10 @@ <h1><a href="/">Census Viz</a>
       <div id="data-table"></div>
     </div>
     <footer>
-      Made by Kyra Acquah, Shuli Jones, and Olivia Tobin.
+      <div>Made by Kyra Acquah, Shuli Jones, and Olivia Tobin.</div>
+      <div id="sign-out-btn"><a href="/" onclick="signOut();">Sign out</a></div>
```

Comment:
Would it be possible to give this button the same styling as the sign in button? If not, no worries

Reply:
Done, currently looks the same except without the Google logo, open to thoughts on whether I should also add the logo

Reply:
Looks great, thanks! I think either way's fine, nbd